config/locales/en.yml

```diff
@@ -55,3 +55,15 @@ en:
               blank: 'Enter your email address'
             address:
               blank: 'Enter your postal address'
+        degree:
+          attributes:
+            type_of_degree:
+              blank: 'Enter the type of your degree'
```

Comment:
I think we should change these messages - if only to be consistent with the style we've used so far.

I'll get some pictures to demonstrate here in a mo.

Reply:
Suggestions on labels/error messages:

![image](https://user-images.githubusercontent.com/429326/62136908-f1282500-b2dc-11e9-8c27-4d8a094d7d58.png)

@paulrobertlloyd do these look better/okay to you?

Reply:
@paulrobertlloyd, we are thinking to add more validations error for each field.
for example: 
- Enter your email
- Format of your email is not correct
Is it ok to show more than one validations error per field? my personal feeling is that the user might be a bit overwhelmed by error messages :), but I may be wrong.

Reply:
The above looks correct (it’s unlikely you would have every field show an error). As to your example:

> * Enter your email
> * Format of your email is not correct

Only one of those errors would be needed, no?

Reply:
my example was bad, but sometimes can happen, I'll tell you more later anyway, not for this branch, thanks!